src/pages/auth/Login.tsx

```diff
@@ -0,0 +1,16 @@
+import { userStore } from '@/store/store';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router';
+
+const Login = () => {
+  const { isAuthenticated } = userStore();
+  const navigate = useNavigate();
+  useEffect(() => {
```

Comment:
@BardhMustafa since now we have this component wrapped with Protected we don't need this part of the code, right?